addons/udes_stock/report/deliveryslip_templates.xml

```diff
@@ -47,69 +62,37 @@
-                            </t>
-
-                            <tr>
-                                <th colspan="2" style="padding-top: 40px" t-if="o.origin">
-                                    <strong>Order Reference</strong>
-                                </th>
-                                <th colspan="2" style="padding-top: 40px" name="td_sched_date_h">
-                                    <strong>Date</strong>
-                                </th>
-                            </tr>
-                            <tr>
-                                <td colspan="2">
-                                    <t t-foreach="set(o.mapped('u_first_picking_ids.origin'))" t-as="origin">
-                                        <strong><span t-esc="origin"/></strong>
-                                    </t>
-                                </td>
-                                <td colspan="2" name="td_sched_date">
-                                   <t t-if="o.state == 'done'">
-                                        <span t-field="o.date_done"/>
-                                   </t>
-                                   <t t-if="o.state != 'done'">
-                                        <span t-field="o.scheduled_date"/>
-                                   </t>
-                                </td>
-                            </tr>
                         </tbody>
                     </table>
 
                     <table class="table table-condensed">  
                         <t t-set="has_serial_number" t-value="o.move_line_ids.mapped('lot_id')" groups="stock.group_production_lot"/>
                         <thead>  
                             <tr>
-                                  <th style="padding-top: 40px"><strong>Pallet</strong></th>
+                                  <th style="padding-top: 40px"><strong>Order Ref.</strong></th>
+                                  <th style="padding-top: 40px"><strong>Pallet No.</strong></th>
                                   <th style="padding-top: 40px"><strong>Product</strong></th>
                                   <th name="lot_serial" t-if="has_serial_number">
                                         Lot/Serial Number
                                   </th>
-                                  <th style="padding-top: 40px"><strong>Ordered Quantity</strong></th>
-                                  <th style="padding-top: 40px"><strong>Sent Quantity</strong></th>
+                                  <th style="padding-top: 40px"><strong>Ordered Qty.</strong></th>
+                                  <th style="padding-top: 40px"><strong>Sent Qty.</strong></th>
                             </tr>
                         </thead>
                         <tbody>    
                             <tr t-if="not o.move_line_ids" t-foreach="o.move_lines" t-as="move">
-                                <td></td>
+                                <td><span t-field="move.origin"/></td>
```

Comment:
Do we need the pallet number here too?

Reply:
I suspect we want it in case we don't have move lines yet but only stock moves. It seems like a safe assumption. That way if you try to print one you get which order has requested what which is good in my view.

Reply:
Not sure if I understood what you meant, but I think if we have no move_lines this should read: 
```suggestion
                                <td><span t-field="move.origin"/></td>
                                <td></td>
```
We need the empty field for the empty pallet number, right?

Reply:
I don't think we need it no, if we have only stock moves then we are displaying stock moves which cannot have pallet so it's redundant. Basically, if your picking is in waiting and you print the delivery slip then you will get this delivery slip which contains the stock moves. Adding a pallet field for this use case doesn't seem necessary in my view.

Reply:
Oh, I see what you're saying. Sorry it wasn't clear. Let me check how it gets displayed.

Reply:
sorry I wasn't being very clear :smile:

Reply:
Yep, it was needed, thank you! I added the field now.